CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+# Contribution guidelines
+
+Ongoing development on this project is managed by this project board: https://github.com/neontribe/www/projects/6.
+
+## Getting tickets/issues ready to be worked on
+
+Before a thing can be worked on it must be both created as an issue, and prioritised using a definition of done.
+
+### Creating issues
+
+1. Open the `Issues` tab and click [`New issue`](https://github.com/neontribe/www/issues/new/choose) and choose the right template.
+   - anyone can create issues
+   - `Bug` is for bugs
+   - `Story` is for new things, e.g. code, content, dependency updates
```

Comment:
```suggestion
   - `Story` is for new things, e.g. code, content to meet a user need
   -  `Tasks` is for simple actions such as dependency updates
```

Reply:
The reason for the edit is that when I was working on this with Rob, I misunderstood the old "task" card and turned it into the "Research" card - I think on reflection we need both.